packages/suite/src/components/suite/PrerequisitesGuide/DeviceRecoveryMode.tsx

```diff
@@ -2,18 +2,13 @@ import { MouseEventHandler } from 'react';
 import { Button } from '@trezor/components';
 import { Translation, TroubleshootingTips } from 'src/components/suite';
 import { rerun } from 'src/actions/recovery/recoveryActions';
-import { useDevice, useDispatch, useSelector } from 'src/hooks/suite';
+import { useDevice, useDispatch } from 'src/hooks/suite';
 
 export const DeviceRecoveryMode = () => {
-    const recovery = useSelector(state => state.recovery);
     const dispatch = useDispatch();
 
     const { isLocked } = useDevice();
 
-    if (recovery.status === 'in-progress') {
```

Comment:
No idea why this was there. Is it safe to remove? It prevents the more-info to show and in the recovery-situation prevents the display of [Continue] button.